include/linux/soundwire/sdw.h

```diff
@@ -914,7 +969,7 @@ struct sdw_bus {
 	int controller_id;
 	unsigned int link_id;
 	int id;
-	int (*compute_params)(struct sdw_bus *bus);
+	int (*compute_params)(struct sdw_bus *bus, struct sdw_stream_runtime *stream);
```

Comment:
@bardliao for the same of keeping this patch simpler to read, can you move this struct definition instead?

Reply:
Yes, done